build/_data/totals.json

```diff
@@ -44,27 +229,25 @@
         "Total_Amount": 40641.0,
         "election_name": "oakland-march-2020"
       }
-    ]
-  },
-  "sf-2016": {
-    "Within California": 100347.02,
-    "largest_independent_expenditures": null
-  },
-  "sf-june-2018": {
-    "Within San Francisco": 6248626.529999998,
-    "Within California": 2219438.2399999998,
-    "Out of State": 14819989.39,
-    "largest_independent_expenditures": null
-  },
-  "sf-2018": {
-    "Out of State": 52025.0,
-    "Within California": 121545.0,
-    "Within San Francisco": 198973.25,
-    "largest_independent_expenditures": [
+    ],
+    "most_expensive_races": [
       {
-        "Filer_NamL": "SAN FRANCISCO APARTMENT ASSOCIATION POLITICAL ACTION COMMITTEE",
-        "Total_Amount": 10177.0,
-        "election_name": "sf-2018"
+        "title": "Measure Q",
+        "type": "referendum",
+        "slug": "oakland-parks-and-recreation-preservation-litter-reduction-and-homelessness-support-act",
+        "amount": 335992.0
+      },
+      {
+        "title": "Measure R",
+        "type": "referendum",
+        "slug": "official-city-newspaper",
+        "amount": 0
```

Comment:
No money for the newspaper referendum I guess. Should we filter out races where there are no contributions? Also - is it right that only Measure Q had any campaign finance data?

Reply:
Yea I think filtering out could make sense. I can add a separate task since that's a good one for a newbie to pick up. As for Measure R and Measure S, looks like we didn't have finance data:

https://www.opendisclosure.io/referendum/oakland/2020-03-03/official-city-newspaper/
https://www.opendisclosure.io/referendum/oakland/2020-03-03/increase-appropriations-limit/